lib/android/app/src/main/java/com/reactnativenavigation/viewcontrollers/viewcontroller/LayoutDirectionApplier.kt

```diff
@@ -0,0 +1,15 @@
+package com.reactnativenavigation.viewcontrollers.viewcontroller
+
+import com.facebook.react.ReactInstanceManager
+import com.facebook.react.modules.i18nmanager.I18nUtil
+import com.reactnativenavigation.options.Options
+
+class LayoutDirectionApplier {
+    fun apply(root: ViewController<*>, options: Options, instanceManager: ReactInstanceManager) {
+        if (options.layout.direction.hasValue() && instanceManager.currentReactContext != null) {
+            root.activity.window.decorView.layoutDirection = options.layout.direction.get()
+            I18nUtil.getInstance().allowRTL(instanceManager.currentReactContext, options.layout.direction.isRtl)
+            I18nUtil.getInstance().forceRTL(instanceManager.currentReactContext, options.layout.direction.isRtl)
+        }
+    }
+}
```

Comment:
Couldn't find any reference to `LayoutDirectionApplier`, do we really need this?

Reply:
idk if we really need this, it is mainly used to change layout direction in order to make all children in the tree rerender with changed directions, you may look at the blame of the old file (the java one, converted while debugging to kotlin)